### PR TITLE
New/tabs

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

**Review by proand — COMMENTED** [review]
This can be as it is until we deal with new default main menu:
image


...but this one should have been kept as it was:
image

Let's discuss it if you disagree...

File: import/_all.scss [hunk 1]
```diff
@@ -4,9 +4,11 @@
 @import "../src/bootstrap-customisations/accordion";
 @import "../src/bootstrap-customisations/alerts";
 @import "../src/bootstrap-customisations/buttons";
+@import "../src/bootstrap-customisations/breadcrumb";
```
Comment (proand) [inline import/_all.scss:7]: breadcrumb?

File: src/blocks/app-and-main-menu/_fhi-main-menu.block.scss [hunk 2]
```diff
@@ -83,19 +88,39 @@
...
+    .nav-link {
+        border-top: 0;
+        border-bottom: $fhi-px * 3 solid transparent;
+        line-height: normal;
+        min-width: $fhi-px * 54;
+        position: relative;
+        text-align: center;
+
+        &.active {
+            &::before {
```
Comment (proand) [inline src/blocks/app-and-main-menu/_fhi-main-menu.block.scss:108]: Nice solution 👍

File: src/blocks/app-and-main-menu/_fhi-main-menu.block.scss [hunk 3]
```diff
@@ -83,19 +88,39 @@
...
+        }
+    }
+}
 
-                &:hover {
-                    background-color: $fhi-blue-dark-2;
-                }
+
+.fhi-main-menu {
+    .nav-link {
```
Comment (proand) [inline src/blocks/app-and-main-menu/_fhi-main-menu.block.scss:99]: Don't style on Bootstrap classes, use BEM

**Review by proand — CHANGES_REQUESTED** [review]
Remove breadcrumb code from this PR

File: src/blocks/app-and-main-menu/_fhi-main-menu.block.scss (hunk 3 above)
Comment (yleaxeman) [reply src/blocks/app-and-main-menu/_fhi-main-menu.block.scss:99]: This is just temporarily. The proper styling will be part of the Global Header PR

File: import/_all.scss (hunk 1 above)
Comment (yleaxeman) [reply import/_all.scss:7]: Since we haven't made a new release, the breadcrumb from previous sprint is fetched from the demo-repo.

Comment (yleaxeman) [reply import/_all.scss]: Should be ok now

**Review by proand — APPROVED** [review]

**Pull request merged**